### ISSUE TITLE
[BUG]: How Orphaned Apps are categorized in "Set App Permission" App?

### ISSUE THREAD

**Issue by dips23** [microsoft/powerapps-tools#395]
**Describe the bug**
How Orphaned Apps are categorized in "Set App Permission"?

**Component (please tell us which flow or app you are experiencing issues with):**
 - Core Solution 
 - CoE Starter Kit - v2.1
 - App - Set App Permissions

**To Reproduce**
1. Go to "Set App Permission" in COE Environment
2. Select environment
3. Click "Orphaned Apps"

**Questions:**
How the "Orphaned Apps" is categorized? Does this list deleted canvas apps only?
What happens to an app, if the app owner's account is removed from Microsoft 365? Will it be listed in this category?

Thank you in advance!

Regards,
Dipti

**Comment by JeneferM-MSFT** [comment]
Yes, that is how orphaned apps/flows are classified. When the person who owned them no longer resolves in the O365 user list. 
This is done in the sync flow for each object type as shown here for apps. (Shown below)
Expand the Upsert App record (abandoned app) step and you will see it being marked orphaned.  

We do this in the sync flows for all the objects we track (flow, model driven apps, canvas, PVA, etc)
This particular app give you the ability to edit the permissions for Canvas Apps.

![image](https://user-images.githubusercontent.com/62659043/88673407-6d467b80-d09d-11ea-9493-d9710dc5feb1.png)

**Comment by dips23** [comment]
Thank you so much Jenefer for the clarification! Much appreciated!

Regards,
Dipti

**Comment by JeneferM-MSFT** [comment]
Closing out issues without furst activity. Thanks for using CoE!
